src/Spectre.libGenetic/Individual.cpp

```diff
@@ -54,28 +54,4 @@ Individual& Individual::operator=(const Individual &other)
     }
 }
 
-std::vector<bool>::reference Individual::operator[](size_t index)
-{
-    if (index < size())
-    {
-        return (*this)[index];
```

Comment:
The infinite recursion problem could be solved [in this way](https://stackoverflow.com/a/34466458).